Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdb/dao/PatientDAO.java

```diff
@@ -453,4 +451,89 @@ public void delete(Patient patientRecord) {
         return patients;
     }
 
+    public Patient readTransaction(long patientId, boolean allRecords) {
```

Comment:
No need for allRecords in this method. allRecords value is used to print only log statements.

Reply:
remove LOG and get-records-size; add-comment to Lazy-loading an object from dataless connection.